api/v1/db/json/notes-dao.js

```diff
@@ -53,7 +53,8 @@ const getNotes = query => new Promise((resolve, reject) => {
 
     const { studentID } = query;
     const studentDirPath = `${dbDirectoryPath}/${studentID}`;
-    const noteFiles = fs.existsSync(studentDirPath) ? fs.readdirSync(studentDirPath) : [];
+    let noteFiles = fs.existsSync(studentDirPath) ? fs.readdirSync(studentDirPath) : [];
+    noteFiles = _.filter(noteFiles, it => it.split('.').pop().toLowerCase() === 'json');
```

Comment:
Instead of parsing extension manually, I think the `path` module can help you save a little works: https://stackoverflow.com/questions/10865347/node-js-get-file-extension

Reply:
Nice, I'll use this instead